qt/applications/workbench/workbench/projectrecovery/projectrecoverysaver.py

```diff
@@ -58,8 +58,14 @@ def recovery_save(self):
         self.pr.thread_on = False
 
         try:
+            # Create directory for save location
```

Comment:
I must be missing something but where was the directory coming from before if it has to be created now?